src/components/Modals/Send/Send.tsx

```diff
@@ -36,11 +37,12 @@ export const SendModal = ({ asset, accountId }: SendModalProps) => {
             <Route
               path='/'
               component={(props: RouteComponentProps) => {
-                // TODO(gomes): make this cleaner
-                if (asset.chain === ChainTypes.Cosmos || asset.chain === ChainTypes.Osmosis) {
-                  return <CosmosForm asset={asset} accountId={accountId} {...props} />
-                }
-                return <Form asset={asset} accountId={accountId} {...props} />
+                const { chainNamespace } = fromChainId(asset.chainId)
+                return chainNamespace === CHAIN_NAMESPACE.Cosmos ? (
```

Comment:
missing osmosis?

Reply:
They share the same `ChainNamespace`,  `cosmos`.